### ISSUE TITLE
Feature request: support for sklearn.preprocessing.OrdinalEncoder

### ISSUE THREAD

**Issue by rwjmiller** [jpmml/jpmml-sklearn#104]
Although there is already support for LabelEncoder and PMMLLabelEncoder, which serve the same purpose, it would nevertheless be useful to have this one as well. There are two advantages of having OrdinalEncoder:

- A single OrdinalEncoder can encode multiple categorical features, allowing for simpler code if you have lots of them.
- Unlike the other two, OrdinalEncoder has the standard sklearn function signature, so it works properly within sklearn.compose.ColumnTransformer and sklearn.pipeline.Pipeline, whereas the other two don't, unless you put them inside a DataFrameMapper or something like that. [This appears to be by design.](https://github.com/scikit-learn/scikit-learn/issues/12720)

I would like to be able to switch from DataFrameMapper to ColumnTransformer in my pipelines, as the latter supports parallel processing and is potentially much faster to fit, but the lack of a supported option for label encoding is currently a deal breaker. If anyone knows of a workaround supported by jpmml-sklearn, I would be interested to hear it.
